src/services/cv_service.py

```diff
@@ -0,0 +1,153 @@
+import PyPDF2
+from datetime import date
+
+
+# Returns the first name from inside the cv
+
+
+def get_first_name_from_cv(cv_content: str) -> str:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Name:' in line:
+            specific_line = line
+
+    first_name = specific_line.split(' ')[1]
+
+    return first_name
+
+
+# Returns the last name from inside the cv
+
+
+def get_last_name_from_cv(cv_content: str) -> str:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Name:' in line:
+            specific_line = line
+
+    last_name = specific_line.split(' ')[2]
+
+    return last_name
+
+
+# Returns the date of birth from inside the cv
+
+
+def get_date_of_birth_from_cv(cv_content: str) -> date:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Geburtsdatum / -ort:' in line:
+            specific_line = line
+
+    date_of_birth = specific_line.split(' ')[3].split('.')
+    day = int(date_of_birth[0])
+    month = int(date_of_birth[1])
+    year = int(date_of_birth[2])
+
+    return date(year, month, day)
+
+
+# Returns the street from inside the cv
+
+
+def get_street_from_cv(cv_content: str) -> str:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Anschrift:' in line:
+            specific_line = line
+
+    street = specific_line.split(' ')[1]
+    number = specific_line.split(' ')[2][:-1]
+
+    return street + ' ' + number
+
+
+# Returns the street from inside the cv
+
+
+def get_street_from_cv(cv_content: str) -> str:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Anschrift:' in line:
+            specific_line = line
+
+    street = specific_line.split(' ')[1]
+    number = specific_line.split(' ')[2][:-1]
+
+    return street + ' ' + number
```

Comment:
These are the exact same.

Reply:
Yes this is because "doppelt hält besser" 🤡. I removed it.